test/e2e/util/util.go

```diff
@@ -621,3 +652,6 @@ func podConditionsReady(pod corev1.Pod) error {
 
 	return nil
 }
+func GenerateClusterRoleBindingName(clusterName string) string {
```

Comment:
one more line above, Is this func a generic func that also have been used in controller?

Reply:
changed use same func with controller.